tests/test_ptycho.py

```diff
@@ -268,6 +274,7 @@ def test_consistent_cgrad(self):
                 'num_gpu': 4,
                 'recover_probe': True,
                 'recover_psi': True,
+                'use_mpi': False,
```

Comment:
```suggestion
                'use_mpi': True,
```
This parameter must be set to True for the mpiexec tests to actually test anything.